### PR TITLE
refactor: cmd/phlare/main.go

### PR DESCRIPTION
**Pull request by AkshayAwate**
Given that identical segments of code are being employed in multiple instances, it is better to utilize the concept of functions, whereby the code is encapsulated within a named block that can be invoked at various points within the program.

### CODE REVIEW

**Review by cyriltovena — APPROVED** [review]
LGTM

**Comment by hi-rustin** [comment]
@cyriltovena I guess you need to approve the first-time contributor's first PR to run the GitHub actions.

File: cmd/phlare/main.go [hunk 1]
```diff
@@ -43,7 +43,12 @@ func (mf *mainFlags) RegisterFlags(fs *flag.FlagSet) {
 	fs.BoolVar(&mf.PrintHelp, "help", false, "Print basic help.")
 	fs.BoolVar(&mf.PrintHelpAll, "help-all", false, "Print help, also including advanced and experimental parameters.")
 }
-
+func errorHandler(testMode bool) {
```
Comment (simonswine) [inline cmd/phlare/main.go:46]: Nit: I find moving testMode a little more elegant

```suggestion
func errorHandler() {
	testMode := cfg.GetTestMode()
```

**Review by simonswine — APPROVED** [review]
LGTM

File: cmd/phlare/main.go (hunk 1 above)
Comment (AkshayAwate) [reply cmd/phlare/main.go]: yes, looks good, made the changes!

**Pull request merged**